client/src/redux/actions/login_actions.js

```diff
@@ -2,87 +2,96 @@ import Actions from './action_types';
 import axios from 'axios';
 
 const API_HOST = 'http://api.meirim.org';
-axios.defaults.withCredentials = true; 
+axios.defaults.withCredentials = true;
 
 const loginActions = {
 
-	login: function (loginForm) {
-		return function (dispatch) {
-			dispatch({
-				type: Actions.LOGIN,
-			});
-			return axios.post(API_HOST + '/sign/in', loginForm).then(function (result) {
+  login: function (loginForm) {
+    return function (dispatch) {
+      dispatch({
+        type: Actions.LOGIN,
+      });
+      return axios.post(API_HOST + '/sign/in', loginForm).then(function (result) {
         window.location.href = '/alerts';
-			}, function (result) {
-				return dispatch({
-					type: Actions.LOGIN_ERROR,
-					data: result
-				});
-			});
-		};
-	},
+      }, function (result) {
+        return dispatch({
+          type: Actions.LOGIN_ERROR,
+          data: result
+        });
+      });
+    };
+  },
 
-	signup: function (loginForm) {
-		return function (dispatch) {
-			dispatch({
-				type: Actions.LOGIN,
-			});
-			return axios.post(API_HOST + '/sign/up', loginForm).then(function (result) {
-				return dispatch({
-					type: Actions.SIGNUP_SUCCESS,
-					data: result
-				});
-			}, function (result) {
-				return dispatch({
-					type: Actions.SIGNUP_ERROR,
-					data: result
-				});
-			});
-		};
-	},
+  signup: function (loginForm) {
```

Comment:
i think a lot of this verbose syntax can really enjoy from shorthand lambdas.. (e.g =>)